edgexfoundry/device-modbus-go
# build(deps)!: bump device-sdk-go and go-mod-core-contracts versions

BREAKING CHANGE: removed the profile update callback REST API

Signed-off-by: Ginny Guan <ginny@iotechsys.com>







**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/device-modbus-go/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

  <link to docs PR>

## Testing Instructions

1. build and run device-modbus with other services
2. create a device with profile Random-Boolean-Device
3. add a new device resource to profile Random-Boolean-Device 
4. verify device callback is successfully triggered by system event:
```
level=DEBUG ts=2023-02-13T06:59:48.205964Z app=device-modbus source=callback.go:56 msg="System event received on message queue. Topic: edgex/system-events/core-metadata/+/+/device-modbus/#, Correlation-id: 871f0112-5768-4a27-aad7-16f74879c62e "
level=DEBUG ts=2023-02-13T06:59:48.206476Z app=device-modbus source=callback.go:40 msg="profile Random-Boolean-Device updated"
```
## New Dependency Instructions (If applicable)
